### PR TITLE
Red/master

### PR DESCRIPTION
**Pull request by RSKYS**

### CODE REVIEW

**Comment by RSKYS** [comment]
From [this message](https://github.com/SlackBuildsOrg/slackbuilds/pull/6817#issuecomment-2177981231):

> audio/tauonmb: tauonmb.SlackBuild failed (status 2)                     07:40:49
> tar: /tmp/SBo/repo.YGNiJ5/slackbuild_tauonmb/kissfft.tar.gz: Cannot open: No such file or directory
> tar: Error is not recoverable: exiting now
> See /var/log/repo/SBo/audio/tauonmb/build.log
> Unmounting chroot ... done.
> ⛔️ audio/tauonmb FAILED ⛔️
> 
> ================================================================================
> SUMMARY                                                                 07:40:50
> ================================================================================
> 
> Logfile:     /var/log/repo/SBo/repo_2024-06-19_07:40:23.log
> Runtime:     0h 0m 28s
> Updated:     3
>   python/python3-plexapi/python3-plexapi-4.15.13-x86_64-1_SBo.txz
>   python/python3-pulsectl/python3-pulsectl-24.4.0-x86_64-1_SBo.txz
>   python/python3-pysdl2/python3-pysdl2-0.9.16-x86_64-1_SBo.txz
> Failed:      1
>   audio/tauonmb
> ```
> 
> ## Snipped Build Failure

According to log sbo-bot failed to read kissfft.tar.gz from current working directory variable path.

**Comment by willysr** [comment]
probably it failed to download

**Comment by willysr** [comment]
@sbo-bot: build audio/tauonmb

**Comment by RSKYS** [comment]
Hmm, this is strange
I've already provided other two for download, it should be at cwd

File: audio/tauonmb/tauonmb.SlackBuild [hunk 1]
```diff
@@ -61,15 +61,24 @@ mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
 rm -rf $SRCNAM-$VERSION
 tar xvf $CWD/$SRCNAM-$VERSION.tar.gz
+
 cd $SRCNAM-$VERSION
+
+# PHAZOR dependencies (Do not modify this part)
+( mkdir -p src/phazor/{kissfft,miniaudio}
```
Comment (willysr) [inline audio/tauonmb/tauonmb.SlackBuild:68]: try removing ( )

**Review by willysr — COMMENTED** [review]
please try that

**Comment by willysr** [comment]
@sbo-bot: build audio/tauonmb

**Comment by willysr** [comment]
if i ran this command: wget --content-disposition https://github.com/mborgerding/kissfft/archive/131.1.0/kissfft.tar.gz
it resulted as kissfft-131.1.0.tar.gz

**Comment by RSKYS** [comment]
Ah, I see.
If you normally wget it without parameters it should download kissfft.tar.gz
Okay I'll fix it

**Comment by willysr** [comment]
@sbo-bot: build audio/tauonmb

**Comment by RSKYS** [comment]
@willysr Can I also add new packages to slackbuildsrepo or I need to submit on website first?
I'm asking this to prevent later confusions, I did it before on these commits:

`22f1fdfc373d7622f5bccb41b9eae4d39f42049b`
`9a3dc9c11a56b3d850da0f193d7145f5288c2467`
`31f6026b398c9a9d9847ab694ba43f091d74c54e`
And...

If I shouldn't do it further, please let me know.
Thanks

**Comment by RSKYS** [comment]
I think the case was that they were depnendency of another app, that's why I could add them alongside updated package, right?

**Comment by willysr** [comment]
> @willysr Can I also add new packages to slackbuildsrepo or I need to submit on website first? I'm asking this to prevent later confusions, I did it before on these commits:
> 
> `22f1fdfc373d7622f5bccb41b9eae4d39f42049b` `9a3dc9c11a56b3d850da0f193d7145f5288c2467` `31f6026b398c9a9d9847ab694ba43f091d74c54e` And...
> 
> If I shouldn't do it further, please let me know. Thanks

you should submit via submission form if it's a new script

**Pull request closed without merging**